elide-model-config/src/main/java/com/yahoo/elide/modelconfig/jsonformats/ElideJoinTypeFormatAttr.java

```diff
@@ -15,15 +15,16 @@
 /**
  * Format specifier for {@code elideJoinType} format attribute.
  * <p>
- * This specifier will check if a string instance is one of {@code ToOne, ToMany}.
+ * This specifier will check if a string instance is one of {@code left, right, inner, outer, cross}.
  * </p>
  */
 public class ElideJoinTypeFormatAttr extends AbstractFormatAttribute {
-    private static final String JOIN_TYPE_REGEX = "^(?i)(ToOne|ToMany)$";
+    private static final String JOIN_TYPE_REGEX = "^(?i)(left|right|inner|outer|cross)$";
```

Comment:
I think we should have left, inner, full, and cross as the join types.

Reply:
done.